### ISSUE TITLE
base64 in css

### ISSUE THREAD

**Issue by RaSeR** [rmrevin/yii2-minify-view#14]
base64 svg and other images in css, minify non correct. but if move it into pure html img tag it pass correctly.

**Comment by rmrevin** [comment]
Show your css style, that is wrong minification.
Just I checked. Base64 png and svg working normally at the minification.

**Comment by RaSeR** [comment]
Im using 1.7.8 and really don't know if you fixed it later.
```css
@font-face {
    src: url(data:font/truetype;charset=utf-8;base64, ... 
}
```
passing ok.
```css
.class {
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANklEQVQImW3JoRHAIAAEwSWKgiIpIqklhdEEHb2JxKAYTu6VJBU3Pjxw4cfAu4aSxFY/YTthnbk4D90EWKnJAAAAAElFTkSuQmCC');
}
```
output fails. without really php errors, just browser cant read property correct. chrome last version.

**Comment by rmrevin** [comment]
Are you sure you have the valid base64?
I could not decrypt `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAANklEQVQImW3JoRHAIAAEwSWKgiIpIqklhdEEHb2JxKAYTu6VJBU3Pjxw4cfAu4aSxFY/YTthnbk4D90EWKnJAAAAAElFTkSuQmCC` through a browser or through an online service.

**Comment by RaSeR** [comment]
sure, cause i move this code to <img> tag and it works fine.

**Comment by rmrevin** [comment]
Fixed in version 1.8.5.
